draft-ietf-tls-dtls13.md

```diff
@@ -471,13 +479,13 @@ unpacked RecordNumber structure, as shown below:
 ~~~
 %%% Record Layer
     struct {
-        uint16 epoch;
+        uint64 epoch;
         uint48 sequence_number;
     } RecordNumber;
 ~~~
 
-This 64-bit value is used in the ACK message as well as in the "record_sequence_number"
-input to the AEAD function.
+This 112-bit value is used in the ACK message. The low order 64 bits are used in
+the "record_sequence_number" input to the AEAD function.
```

Comment:
So I don't think that this is quite enough.  Technically, it's fine, but "record_sequence_number" is not a thing.  I couldn't find a definition in this doc or RFC 8446.  Why not just cite [Section 5.3 of RFC 8446](https://datatracker.ietf.org/doc/html/rfc8446#section-5.3)?

Reply:
That's existing text, but we can certainly improve it here.

Reply:
@martinthomson - done.